recipe/meta.yaml

```diff
@@ -1,51 +1,130 @@
+  - url: https://gstreamer.freedesktop.org/src/gst-plugins-base/gst-plugins-base-{{ version }}.tar.xz
+    sha256: 1026c7c3082d825d9b5d034c1a6dd8a4ebab60eb3738b0a0afde4ad2dc0b0db5
+    folder: plugins_base
 
 build:
-  skip: true  # [not linux]
+  skip: true  # [win]
   number: 0
-  run_exports:
-    # remove symbols at .90 patch releases just before minor versions?
-    #    https://abi-laboratory.pro/tracker/timeline/gstreamer/
-    - {{ pin_subpackage('gstreamer', max_pin='x.x') }}
-
-requirements:
-  build:
-    - {{ compiler('c') }}
-    - bison
-    - flex
-    - m4
-    - perl
-  host:
-    - glib
-    - xz
-  run:
-    - glib
-
-test:
-  commands:
-    - gst-inspect-1.0 --version
-    - gst-launch-1.0  --version
-    - gst-stats-1.0 --version
-    - gst-typefind-1.0 --version
+
+outputs:
+  - name: gstreamer
+    script: install_gstreamer.sh
+    build:
+      activate_in_script: True
+      run_exports:
+        # remove symbols at .90 patch releases just before minor versions?
+        #    https://abi-laboratory.pro/tracker/timeline/gstreamer/
+        - {{ pin_subpackage('gstreamer', max_pin='x.x') }}
+
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - pkg-config
+        - bison
+        - flex
+        - m4
+        - autoconf
+        - automake
+        - make
+        - perl
+      host:
+        - gobject-introspection
+        - gettext                            # [osx]
+        - pcre                               # [linux]
+        - libffi                             # [linux]
+        - glib
+        - xz
+    test:
+      commands:
+        - gst-inspect-1.0 --version
+        - gst-launch-1.0  --version
+        - gst-stats-1.0 --version
+        - gst-typefind-1.0 --version
+    about:
+      summary: "Library for constructing graphs of media-handling components"
+      description:
+      doc_source_url: https://cgit.freedesktop.org/gstreamer/gstreamer/tree/docs
+
+  - name: gst-plugins-base
+    script: install_base_plugins.sh
+    build:
+      activate_in_script: True
+      run_exports:
+        # remove symbols at .90 patch releases just before minor versions?
+        #    https://abi-laboratory.pro/tracker/timeline/gstreamer/
+        - {{ pin_subpackage('gst-plugins-base', max_pin='x.x') }}
+
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - {{ cdt('libxau-devel') }}          # [linux]
+        - {{ cdt('libxext-devel') }}         # [linux]
+        - {{ cdt('libx11-devel') }}          # [linux]
+        - {{ cdt('libxrender-devel') }}      # [linux]
+        - {{ cdt('xorg-x11-proto-devel') }}  # [linux]
+        - {{ cdt('mesa-libgl-devel') }}      # [linux]
+        - {{ cdt('libdrm-devel') }}          # [linux]
+        - {{ cdt('libxdamage-devel') }}      # [linux]
+        - {{ cdt('libxfixes-devel') }}       # [linux]
+        - {{ cdt('libxxf86vm-devel') }}      # [linux]
```

Comment:
@msarahan do you think you could talk about the `cdt` usage a little bit in our next meeting?